.github/workflows/ci.yml

```diff
@@ -35,10 +35,52 @@ jobs:
           opam pin add . -y --no-action
           opam depext -y geneweb
           opam install -y ./*.opam --deps-only --with-test
-          opam pin ocamlformat 0.24.1
       - name: Make ocamlformat > build/distrib
         run: |
           opam exec -- ocaml ./configure.ml --release
-          opam exec -- make fmt distrib
```

Comment:
We probably want to keep the formatting check and thus, the dependency on `ocamlformat` for the developers.